src/common/offline_transformations/src/compress_quantize_weigths.cpp

```diff
@@ -357,7 +357,7 @@ static void compute_scale_and_zero_point_internal(const std::shared_ptr<ov::op::
                                                                        float output_high_value) mutable {
             float output_range = output_high_value - output_low_value;
             float scale = output_range / input_range;
-            float zero_point_value = (new_output_low - output_low_value / scale) * (scale != 0);
+            float zero_point_value = (scale != 0) ? (new_output_low - output_low_value / scale) : 0;
```

Comment:
Question:
Is it ok to compare a float variable with 0? Should it be a comparison with an epsilon?

Reply:
Should be ok. Dividing by very small, yet non zero floating point value does not yield inf or nan.